### DISCUSSION TITLE
Firefox 122 Ubuntu, CoApp is broken, Work In Progress

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#12] [Announcements]
The latest Firefox update on Ubuntu seems to have broken the Companion App.

We are working on it.

**Comment by paulrouget** [comment]
Filed a bug: https://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+bug/2051188

I'll keep investigating.

**Comment by paulrouget** [comment]
Firefox bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1876447

**Comment by paulrouget** [comment]
Bug has been found. We're waiting on Canonical to fix that on their end, and hopefully they will update the Snap package soon.

**Comment by paulrouget** [comment]
From canonical:

> Thanks for the bug report. The messaging portal patch is indeed part of the snap and not upstream at this point. The patch was refreshed to apply cleaning to 122 and rebased at the same time on the current phabricator version which made it use a new dbus method. We didn't notice that this method hadn't been backported to the Ubuntu 22.04 portal package. We are working now on reverting the patch to not depends on the new method.

**Comment by paulrouget** [comment]
Bug has been fixed by Canonical. Please update your Ubuntu install.

**Comment by Paolo59it** [comment]
After updating Firefox to the last snap version, that Ubuntu proposed today, the problem is solved.

**Reply by Len-Palmeri** [reply]
I tried "sudo snap refresh firefox"  but it says:   snap "firefox" has no updates available.

**Reply by paulrouget** [reply]
Also make sure to update Snap Core.

**Reply by Len-Palmeri** [reply]
CoApp is back in action here!
Thanks so much Paul for keeping us all updated on this issue.
